connectors/openai/element-templates/openai-connector.json

```diff
@@ -0,0 +1,330 @@
+  },
+  "documentationRef": "https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/rest/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "groups": [
+    {
+      "id": "authentication",
+      "label": "Authentication"
+    },
+    {
+      "id": "input",
+      "label": "Payload"
+    },
+    {
+      "id": "input",
+      "label": "Payload"
+    },
+    {
+      "id": "output",
+      "label": "Response Mapping"
+    },
+    {
+      "id": "errors",
+      "label": "Error Handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "id": "authenticationType",
+      "value": "bearer",
+      "type": "Hidden",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      }
+    },
+    {
+      "label": "OpenAI API Key",
```

Comment:
For how long does it last? If it's a typical bearer token that lasts only 3600 seconds, maybe it's worth to take a look whether client credentials flow is supported by OpenAI

Reply:
It's a [static API key](https://platform.openai.com/docs/api-reference/authentication) - but for some reason they call it a bearer token.

BTW I just noticed that it makes sense to add [organization header](https://platform.openai.com/docs/api-reference/requesting-organization) support.

Reply:
Thanks for confirming. That's weird, but I see it as well now, so no issues here.